config/passport.js

```diff
@@ -0,0 +1,137 @@
+// config/passport.js
+
+// load all the things we need
+var LocalStrategy   = require('passport-local').Strategy;
+var FacebookStrategy = require("passport-facebook").Strategy;
+
+// load up the user model
+var mysql = require('mysql');
+var bcrypt = require('bcrypt-nodejs');
+var dbconfig = {
+    host: process.env.DATABASE_HOST,
+    user: process.env.DATABASE_USER,
+    password: process.env.DATABASE_PASSWORD
+};
+var connection = mysql.createConnection(dbconfig);
+
+var db = require("../models");
+
+connection.query('USE ' + process.env.DATABASE_DB);
```

Comment:
Not sure if this is technically necessary now that I am using the ORM. Will test tomorrow.